tuf/repository_lib.py

```diff
@@ -92,11 +92,15 @@
 SUPPORTED_KEY_TYPES = ['rsa', 'ed25519', 'ecdsa-sha2-nistp256']
 
 
+
+
 def _generate_and_write_metadata(rolename, metadata_filename,
   targets_directory, metadata_directory, storage_backend,
   consistent_snapshot=False, filenames=None, allow_partially_signed=False,
   increment_version_number=True, repository_name='default',
-  use_existing_fileinfo=False):
+  use_existing_fileinfo=False,use_timestamp_length=True,
```

Comment:
missing a space after the `,`
```suggestion
  use_existing_fileinfo=False, use_timestamp_length=True,
```